conjure_oxide/src/main.rs

```diff
@@ -28,32 +31,43 @@ struct Cli {
 }
 
 pub fn main() -> AnyhowResult<()> {
+    let log_file = File::options()
+        .create(true)
+        .append(true)
+        .open("conjure_oxide.log")
+        .unwrap();
+
+    Builder::new()
+        .with_target_writer("info", new_writer(stdout()))
+        .with_target_writer("file", new_writer(log_file))
```

Comment:
Is whether it goes to stout or log_file dependant on the target= variable, or is it like python where (eg) warnings and above go to console by default

Reply:
It's the target= parameter so for each log call you can set the output method, apologies for not clarifying!

Reply:
My instinct is that people writing logs shouldn't be choosing between 'info' and 'file' -- they should choose an importance level, and then the importance level can decide where the event goes (screen, file, or nowhere).

I'd be tempted (open to debate however!) to change the target_writer to (I hopefully get the notation right)

.with_target_writer("*", new_writer(log_file))

Then encourage people to use the `target` for the subsystem the warning is coming from, which I have found useful in the past, as sometimes I just want to look at the log from a single subsystem (which you can just do with `grep`/`rg` from the command line, or do something more clever which changes how the log is produced :) )